great_expectations/execution_engine/pandas_execution_engine.py

```diff
@@ -31,8 +30,11 @@
 
 try:
     import boto3
+    from botocore.exceptions import ClientError, ParamValidationError
```

Comment:
@Shinnnyshinshin Do we need to import `botocore` first to insure that the reference to it exist?  Thanks.

Reply:
I think this is ok. Similar patterns exist in `/data_context/util.py` for boto3 and botocore, as well as `google` packages.